Added/Improved Intros for the Map Topics (#6887)
github/docs: content/github/getting-started-with-github/getting-started-with-git/index.md

---
title: Getting started with Git

> intro: '
> Git is a distributed Version Control System(VCS). Get started setting up Git to manage and control all your {% data variables.product.prodname_dotcom %} project workflows locally.'

Reviewer: Let's limit to this one sentence. I also switched "locally" with the more beginner-friendly "from your computer."

```suggestion
intro: 'Set up Git, a distributed version control system, to manage your {% data variables.product.product_name %} repositories from your computer.'
```
